### PR TITLE
Add Ubuntu 20.04 for QAM

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

This PR adds running of Ubuntu 20.04 to QAM test suite.

## Links

Fixes SUSE/spacewalk#12765.

Ports:
* 4.1: SUSE/spacewalk#12787
* 4.0: SUSE/spacewalk#12788
* 3.2: no QAM

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

### CODE REVIEW

File: testsuite/features/qam/add_custom_repositories/add_ubuntu2004_repositories.feature [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@ubuntu2004_minion
```
Comment (calancha) [inline testsuite/features/qam/add_custom_repositories/add_ubuntu2004_repositories.feature:4]: We might want to register the new tags here `testsuite/features/support/env.rb`
Same for `ubuntu2004_sshminion` tag.

Comment (Bischoff) [reply testsuite/features/qam/add_custom_repositories/add_ubuntu2004_repositories.feature:4]: oops, yes

**Comment by srbarrios** [comment]
We stopped porting QAM improvements in 3.2, as we will not support 3.2 for QAM.

**Comment by srbarrios** [comment]
- [ ] Add new tags in env.rb
- [ ] Add new packages, channels, and package arch in constants.rb

**Review by ktsamis — APPROVED** [review]
It's a huge PR, nothing jumps out at me as wrong but I'm not sure 100%. My suggestion, try it and see if it works

**Review by calancha — APPROVED** [review]
LGTM.

**Comment by Bischoff** [comment]
> It's a huge PR, nothing jumps out at me as wrong but I'm not sure 100%. My suggestion, try it and see if it works

@ktsamis most of it is copy and paste... that's why it's huge :-)

**Pull request merged**